io.github.gen2brain.cbconvert.yml

```diff
@@ -0,0 +1,369 @@
+app-id: io.github.gen2brain.cbconvert
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+
+command: cbconvert-gui
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+
+build-options:
+  env:
+    - GOROOT=/usr/lib/sdk/golang
+
+cleanup:
+  - '*.a'
+  - '*.la'
+  - '/bin/Magick*'
+  - '/etc'
+  - '/include'
+  - '/lib/cmake'
+  - '/lib/pkgconfig'
+  - '/lib/ImageMagick*'
+  - '/share'
```

Comment:
this delete things it shouldn't. including the desktop file, etc.

Reply:
Thanks, I thought that cleanup was for the build directory only, if you mean `/share` is wrong. What about the id that ends with `.cbconvert` and the command is `cbconvert-gui`, I think I saw that is a problem in some PRs.

Reply:
> thought that cleanup was for the build directory only,

it's never  about it.

it's all about installed files.